recipes/autoconf-archive/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
```

Comment:
For uniformity with all other recipes.
```suggestion
from conans import ConanFile, tools, AutoToolsBuildEnvironment
import os
```